CHANGELOG.md

```diff
@@ -32,13 +32,17 @@
 * Fixed sorting order of strings to use standard unicode codepoint order instead of grouping similar English letters together. A noticeable change will be from "aAbBzZ" to "ABZabz". ([realm/realm-core#2573](https://github.com/realm/realm-core/issues/2573))
 * `data` and `string` are now strongly typed for comparisons and queries. This change is especially relevant when querying for a string constant on a Mixed property, as now only strings will be returned. If searching for `data` is desired, then that type must be specified by the constant. In RQL the new way to specify a binary constant is to use `mixed = bin('xyz')` or `mixed = binary('xyz')`. ([realm/realm-core#6407](https://github.com/realm/realm-core/issues/6407))
 
+* Fixed diverging history due to a bug in the replication code when setting default null values (embedded objects included) ([realm/realm-core#7536](https://github.com/realm/realm-core/issues/7536)).
+* Null pointer exception may be triggered when logging out and async commits callbacks not executed ([realm/realm-core#7434](https://github.com/realm/realm-core/issues/7434), since v12.6.0)
```

Comment:
```suggestion
* Null pointer exception may be triggered when logging out and async commits callbacks not executed. ([realm/realm-core#7434](https://github.com/realm/realm-core/issues/7434), since v12.6.0)
```